easybuild/easyconfigs/b/bigdft-suite/bigdft-suite-1.9.1-foss-2021b.eb

```diff
@@ -0,0 +1,102 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Denis Kristak
+easyblock = "CmdCp"
+
+name = "bigdft-suite"
+version = "1.9.1"
+
+homepage = "https://gitlab.com/l_sim/bigdft-suite"
+description = """
+BigDFT: electronic structure calculation based on Daubechies wavelets.
+bigdft-suite is a set of different packages to run bigdft.
+"""
+
+toolchain = {"name": "foss", "version": "2021b"}
+toolchainopts = {
+    "usempi": True,
+    "extra_fflags": "-fallow-argument-mismatch",
+    "extra_fcflags": "-fallow-argument-mismatch",
+    "extra_f90flags": "-fallow-argument-mismatch",
+}
+
+source_urls = ["https://gitlab.com/l_sim/bigdft-suite/-/archive/%(version)s/"]
+sources = ["%(name)s-%(version)s.tar.gz"]
+patches = ["bigdft_suite_index_bug.patch"]
+checksums = [
+    "3c334da26d2a201b572579fc1a7f8caad1cbf971e848a3e10d83bc4dc8c82e41",  # bigdft-suite-1.9.1.tar.gz
+    "81d12edb2319685ede822ca82594ea8278850e52370ced5ea40c88fe17187f8c",  # bigdft_suite_index_bug.patch
+]
+
+local_configopts = " ".join(
+    [
+        'CC=\\"$MPICC\\" FC=\\"$MPIFC\\" --enable-mpi',
+        'FCFLAGS=\\"$CPPFLAGS $FCFLAGS -std=legacy\\"',
+        'FFLAGS=\\"$CPPFLAGS $FFLAGS -std=legacy\\"',
+        'F90FLAGS=\\"$CPPFLAGS $F90FLAGS -std=legacy\\"',
+        '--with-ext-linalg=\\"-L$SCALAPACK_LIB_DIR $LIBSCALAPACK -L$LAPACK_LIB_DIR $LIBBLAS\\"',
+        '--with-yaml-path=\\"$EBROOTLIBYAML\\"',
+    ]
+)
+
+local_comp_cmd = (
+    """
+    mkdir bigdft_builddir && \
+    cd bigdft_builddir && \
+    python3 ../Installer.py build bigdft -y -v -c %s """
+    % local_configopts
+)
+
+cmds_map = [(".*", local_comp_cmd)]
```

Comment:
Why not use `Binary` easyblock with `extract_sources = True and `install_cmd = local_comp_cmd` (or just `install_cmd = ' && '.join([...])`?